FakebookNotifications/FakebookNotifications.Testing/MongoDbTests.cs

```diff
@@ -0,0 +1,255 @@
+﻿using FakebookNotifications.DataAccess;
+using FakebookNotifications.DataAccess.Models;
+using FakebookNotifications.DataAccess.Models.Interfaces;
+using Microsoft.Extensions.Options;
+using Moq;
+using MongoDB.Driver;
+using System;
+using Xunit;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace FakebookNotifications.Testing
+{
+    public class MongoDbTests
+    {
+        private Mock<IOptions<NotificationsDatabaseSettings>> _mockSettings;
+        private Mock<IMongoDatabase> _mockDB;
+        private Mock<IMongoClient> _mockClient;
+        private NotificationsDatabaseSettings settings;
+
+        //Constructor to initialize mocked components before each test
+        public MongoDbTests()
+        {
+            _mockSettings = new Mock<IOptions<NotificationsDatabaseSettings>>();
+            _mockDB = new Mock<IMongoDatabase>();
+            _mockClient = new Mock<IMongoClient>();
+
+            settings = new NotificationsDatabaseSettings()
+            {
+                ConnectionString = "mongodb://localhost:27017",
```

Comment:
why are we providing a value here that seems to depend on a db running for our unit tests? or if this value isnt being used, then why set it?